packages/cli/test/functional/test_site/expected/bugs/index.html

```diff
@@ -216,6 +216,9 @@ <h1 id="heading-in-footer-should-not-be-indexed"><span id="heading-in-footer-sho
     </div>
   </div>
 </body>
+
+<script src="/test_site/headFiles/customScriptTop.js"></script>
```

Comment:
this one's a feature https://markbind-master.netlify.app/userguide/tweakingthepagestructure#inserting-content-into-the-head

Reply:
Thanks for the spot!